pkg/api/types.go

```diff
@@ -1391,7 +1391,10 @@ const (
 	ResourceMemory ResourceName = "memory"
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
-	// Number of Pods that may be running on this Node: see ResourcePods
+	// Bandwidth available for egress from a pod (bits/second)
+	ResourceEgressBandwidth = "egress_bandwidth"
```

Comment:
We don't have underscores in the API, and this shouldn't pass validation. I suggest just egress and ingress. Qualities such as latency aren't consumable resources.

cc @thockin @erictune